themes/Atomize-color-theme.json

```diff
@@ -118,6 +118,9 @@
 		"extensionButton.prominentForeground": "#FFFFFF",
 		"extensionButton.prominentHoverBackground": "#5B82D4",
 
+		// Git
+		"gitDecoration.ignoredResourceForeground": "#9DA5B499",
```

Comment:
Do you mind using `#9DA5B599` instead of `#9DA5B499` here? We already use `#9DA5B5` in a couple of other places, so it would be nice to stay as consistent as possible.

Reply:
Atom seems to be using #9DA5B4. Maybe would be better to use that everywhere?

![image](https://user-images.githubusercontent.com/16735302/49204755-8b824280-f3b5-11e8-94e5-20259e993705.png)